### PR TITLE
Address issue #103 and #102

### LINKED ISSUE #102: IndexError when parsing certain .rpt files
I am using the following code to read and parse the attached .inp and .rpt files:
```
model = swmmio.Model(inp_file_name)
elem_dict = {element: model.__getattribute__(element).geojson for element in element_types}
```

When running this code, I get the following error:
```
  File "/home/jolson/.virtualenvs/rest-api/lib/python3.6/site-packages/swmmio/utils/dataframes.py", line 89, in dataframe_from_rpt
    index_col=0, names=cols)
  File "/home/jolson/.virtualenvs/rest-api/lib/python3.6/site-packages/pandas/io/parsers.py", line 686, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/jolson/.virtualenvs/rest-api/lib/python3.6/site-packages/pandas/io/parsers.py", line 458, in _read
    data = parser.read(nrows)
  File "/home/jolson/.virtualenvs/rest-api/lib/python3.6/site-packages/pandas/io/parsers.py", line 1186, in read
    ret = self._engine.read(nrows)
  File "/home/jolson/.virtualenvs/rest-api/lib/python3.6/site-packages/pandas/io/parsers.py", line 2186, in read
    values = data.pop(self.index_col[i])
IndexError: list index out of range
```

It seems like the `dataframe_from_rpt` method has an issue parsing certain kinds of .rpt files. @jennwuu came up with some code to patch this and knows more about why it breaks.

I had to upload the files with a .txt extension to make GitHub happy.

[example.rpt.txt](https://github.com/aerispaha/swmmio/files/5271557/example.rpt.txt)
[example.inp.txt](https://github.com/aerispaha/swmmio/files/5271559/example.inp.txt)

### LINKED ISSUE #103: coords column is not updated in subcatchment dataframe in ModelSection object when index is integer instead of string
When subcatchment name is called (1, 2, 3, 4) the polygon index is automatically converted to integer from string. 
See example model input.

### PR DESCRIPTION
**Pull request by jennwuu**
Closes #103 
Identified an issue when subcatchments are named numerically (e.g. 1, 2, 3, 4), the polygon dataframe index is automatically converted to integer from string, so when coords column is assigned is not updated.

See example_1 model input.

Closes #102 
Add flexibility in columns depending on swmm version number.

I add function to extract swmm version from .rpt file
https://github.com/jennwuu/swmmio/blob/cd03ae9b7cb1c7358b62e0b2d9259ab90c746f7b/swmmio/core.py#L459-L467

I add a new section in header yaml to allow more columns based on swmm version

https://github.com/jennwuu/swmmio/blob/cd03ae9b7cb1c7358b62e0b2d9259ab90c746f7b/swmmio/utils/text.py#L216-L224

```yaml
swmm5_version:
  1.13:
    rpt_sections:
      Subcatchment Runoff Summary:
        - Name
        - TotalPrecip
        - TotalRunon
        - TotalEvap
        - TotalInfil
        - ImpervRunoff
        - PervRunoff
        - TotalRunoffIn
        - TotalRunoffMG
        - PeakRunoff
        - RunoffCoeff
```

### CODE REVIEW

**Comment by jennwuu** [comment]
@aerispaha, not sure why travis fails though

File: swmmio/tests/data/Example6.rpt [hunk 1]
```diff
@@ -0,0 +1,192 @@
+
```
Comment (aerispaha) [inline swmmio/tests/data/Example6.rpt:1]: Can we remove this file. it doesn't seem to be used anywhere in the unit tests

File: swmmio/tests/data/__init__.py [hunk 2]
```diff
@@ -25,6 +25,9 @@
 MODEL_MOD_GREEN_AMPT = os.path.join(DATA_PATH, 'model_mod_green_ampt.inp')
 MODEL_CURVE_NUMBER = os.path.join(DATA_PATH, 'model_curve_num.inp')
 MODEL_MOD_HORTON = os.path.join(DATA_PATH, 'model_mod_horton.inp')
+MODEL_EX_1 = os.path.join(DATA_PATH, 'Example1.inp')
+MODEL_EX_1B = os.path.join(DATA_PATH, 'Example1b.inp')
+MODEL_EX_6 = os.path.join(DATA_PATH, 'Example6.inp')
```
Comment (aerispaha) [inline swmmio/tests/data/__init__.py:30]: let's remove `MODEL_EX_6` unless its being used somewhere

File: swmmio/utils/text.py [hunk 3]
```diff
@@ -211,7 +212,19 @@ def get_inp_sections_details(inp_path, include_brackets=False):
     return found_sects
 
 
-def get_rpt_sections_details(rpt_path):
+
+def update_report_header(swmm_version, rpt_headers):
```
Comment (aerispaha) [inline swmmio/utils/text.py:216]: Can we rename this function to  `update_report_header` ?

**Review by aerispaha — CHANGES_REQUESTED** [review]
I like this a lot. I have a few comments and I'm going to push a couple changes momentarily. Thanks!

**Review by aerispaha — APPROVED** [review]

**Pull request merged**